configs/cassandra.textproto

```diff
@@ -0,0 +1,362 @@
+    "    existence:{should_exist: true}"
+    "    non_compliance_msg: \"Cassandra configuration file does not exist\""
+    "  }"
+    "  file_checks:{"
+    "    files_to_check:{single_file:{path:\"/etc/cassandra/cassandra.yaml\"}}"
+    "    content_entry: {"
+    "      match_type: NONE_MATCH"
+    "      match_criteria: {"
+    "        filter_regex: \"authorizer: .*\""
+    "        expected_regex: \"authorizer: AllowAllAuthorizer\""
+    "      }"
+    "    }"
+    "    non_compliance_msg: \"Cassandra is configured without proper authorization\""
+    "  }"
+    "}"
+  }
+}
+
+benchmark_configs: {
+  id: "cassandra-internodeencryption"
+  compliance_note: {
+    version: { version: "1.0.0" benchmark_document: "CIS Apache Cassandra" }
+    title: "Inter-node Encryption"
+    description:
+      "Cassandra offers the option to encrypt data in transit between nodes on the cluster. By "
+      "default, inter-node encryption is turned off. "
+    rationale:
+      "Data being transferred on the wire should be encrypted to avoid network snooping, whether legitimate or not."
+    remediation:
+      "The inter-node encryption should be implemented before anyone accesses the Cassandra server."
+    cis_benchmark: {
+      profile_level: 1
+      severity: HIGH
+    }
+    scan_instructions:
+    "check_alternatives:{"
+    "  file_checks:{"
+    "    files_to_check:{single_file:{path:\"/etc/cassandra/cassandra.yaml\"}}"
+    "    existence:{should_exist: true}"
+    "    non_compliance_msg: \"Cassandra configuration file does not exist\""
+    "  }"
+
+    "  file_checks:{"
+    "    files_to_check:{single_file:{path:\"/etc/cassandra/cassandra.yaml\"}}"
+    "    content_entry: {"
+    "      match_type: NONE_MATCH"
+    "      match_criteria: {"
+    "        filter_regex: \"\\\\s*internode_encryption:.*\""
+    "        expected_regex: \"\\\\s*internode_encryption:\\\\s*none\""
+    "      }"
+    "    }"
+    "    non_compliance_msg: \"Cassandra does not encrypt internode communication\""
+    "  }"
+    "}"
+  }
+}
+
+benchmark_configs: {
+  id: "cassandra-logginglevel"
+  compliance_note: {
+    version: { version: "1.0.0" benchmark_document: "CIS Apache Cassandra" }
+    title: "Ensure that Cassandra only listens for network connections on authorized interfaces"
+    description:
+      "Apache Cassandra uses Logback for logging functionality. While this can be set using "
+      "nodetool setlogginglevel changes made using this method will be reverted to the level "
+      "specified in the logback.xml file the next time the process restarts."
+    rationale:
+      "If logging is not enabled, issues may go undiscovered, and compromises and other "
+      "incidents may occur without being quickly detected. It may also not be possible to provide "
+      "evidence of compliance with security laws, regulations, and other requirements."
+    remediation:
+      "Enable the Cassandra Logging setting it's logging level to INFO."
```

Comment:
Why not use the remediation text from the benchmark document here? The v1.0.0 one at least describes the file to edit as well.